# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/client/webconsole/test/browser/test-blank-with-cookie.html
# Commit: ff5ea82f6090
# Full Hash: ff5ea82f6090a4a1645a1a32749a4ae9102f015b
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-04-17 16:30:04
# Regressor Bug: 1957355
# File Overlap Count: 1
# Description:
#   Bug 1957355 - No storage access for resources injected by content-scripts, r=timhuang,necko-reviewers,devtools-reviewers,anti-tracking-reviewers,nchevobbe,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D244204
# ==============================================================================

diff -r 04eed8759454 -r ff5ea82f6090 devtools/client/webconsole/test/browser/test-blank-with-cookie.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/devtools/client/webconsole/test/browser/test-blank-with-cookie.html	Thu Apr 17 10:56:59 2025 +0000
@@ -0,0 +1,10 @@
+<!-- Any copyright is dedicated to the Public Domain.
+     http://creativecommons.org/publicdomain/zero/1.0/ -->
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Blank</title>
+</head>
+<body>
+</body>
+</html>